requirements.txt

```diff
@@ -13,6 +13,7 @@ mock>=2.0.0
 PyDOE >= 0.3.0
 sobol_seq >=0.1
 emcee==2.2.1
+torch==1.1.0
```

Comment:
See general comments above

Reply:
There is a reason why I am using PyTorch instead of Sobol, and it is because if I use sebol_seq after 40 iterations it will crash, as it cannot create higher than 40-dimensional sequences. 
Pytorch is the only package I tested that goes above 40.
The only way to do it without PyTorch (or another package that allows a low discrepancy sequence with more than 40 dimensions)  is to just do normal Montecarlo integration, but it would be even slower then it is right now

Reply:
How critical is that funcitonality to the algo? Are there other packages that do similar thing? I would really like us not to bring in PyTorch as a dependecy.

Reply:
I couldn't find any package, I can see if I keep looking for it if I can find one.. it's kind of important to speed up the algorithm, because it's pretty slow already..
I'll see if I can find some other package to do that